eclipse-scout-core/src/security/AccessControl.ts

```diff
@@ -63,10 +67,41 @@ export class AccessControl extends PropertyEventEmitter implements AccessControl
 
   startSync() {
     this._sync();
+    this._subscribeForNotifications();
   }
 
   stopSync() {
-    clearTimeout(this._syncTimeoutId);
+    this._unsubscribeFromNotifications();
+  }
+
+  protected _subscribeForNotifications() {
+    uiNotifications.subscribe('permissionsUpdate', this._permissionUpdateEventHandler);
+  }
+
+  protected _unsubscribeFromNotifications() {
+    uiNotifications.unsubscribe('permissionsUpdate', this._permissionUpdateEventHandler);
+  }
+
+  protected _onPermissionUpdateNotify(event: UiNotificationEvent) {
+    let reloadDelayWindow = scout.nvl(event.message['reloadDelayWindow'], 0) as number;
```

Comment:
Better create PermissionUpdateNotificationDo.ts which contains reloadDelayWindow as typed member to make clear what notification is expected